src/test/java/net/sf/jabref/bibtex/BibtexEntryAssert.java

```diff
@@ -133,21 +132,13 @@ public static void assertEquals(ImportFormat importer, InputStream shouldBeIs, I
      * @param actualEntries the list with the actual entries
      */
     public static void assertEquals(List<BibEntry> shouldBeIs, List<BibEntry> actualEntries) {
-        Assert.assertNotNull(shouldBeIs);
-        Assert.assertNotNull(actualEntries);
-        Assert.assertEquals(shouldBeIs.size(), actualEntries.size());
-        for (int i = 0; i < actualEntries.size(); i++) {
-            assertEquals(shouldBeIs.get(i), actualEntries.get(i));
-        }
+        Assert.assertEquals(shouldBeIs, actualEntries);
     }
 
     /**
-     * Compares to BibTeX entries using their canonical representation
+     * Compares to BibTeX entries
      */
     public static void assertEquals(BibEntry shouldBeEntry, BibEntry entry) {
-        // use the canonical string representation to compare the entries
-        String shouldBeEntryRepresentation = CanonicalBibtexEntry.getCanonicalRepresentation(shouldBeEntry);
-        String entryRepresentation = CanonicalBibtexEntry.getCanonicalRepresentation(entry);
-        Assert.assertEquals(shouldBeEntryRepresentation, entryRepresentation);
+        Assert.assertEquals(shouldBeEntry, entry);
```

Comment:
When there are non equal entries, how do I see the differences easily? With the old implementation, I could see the differences in string representations. What about the new way?